src/dbSta/src/CMakeLists.txt

```diff
@@ -72,6 +72,7 @@ target_sources(dbSta
 target_include_directories(dbSta
   PUBLIC
     ../include
+    ${PROJECT_SOURCE_DIR}/include
```

Comment:
Aren't these the same?  I'm not sure why the new line is needed.

Reply:
${PROJECT_SOURCE_DIR} pull in the OpenRoad.hh files. Which were missing in dbSta